Serializer/Jms/Handler/ResourceHandler.php

```diff
@@ -85,6 +86,7 @@ private function doSerializeResource(Resource $resource, $depth = 0)
         }
 
         $data['path'] = $resource->getPath();
+        $data['node_name'] = PathHelper::getNodeName($data['path']);
```

Comment:
actually, we need both:

* `node_name` is used to build the path in the tree
* `resource_label` is used in the tree display, to make it nicer for admins